src/QuadForm/Types.jl

```diff
@@ -444,7 +444,7 @@ mutable struct LineEnumCtx{T, S}
   dim::Int
   depth::Int
   v::Vector{S}
-  length::BigInt
+  length::IntegerUnion
```

Comment:
Doing this means you just made anything use `length` type unstable, unless you add a type assertion or a type conversion to every place that accesses `length`. Perhaps that's OK, perhaps not -- you could avoid it by adding the type of `length` as another type parameter to `LineEnumCtx`.